modules/inflowwind/src/InflowWind_Subs.f90

```diff
@@ -120,192 +120,64 @@ MODULE InflowWind_Subs
-   IF (ErrStat >= AbortErrLev) THEN
-      CALL Cleanup()
-      RETURN
-   END IF
-
-
-      ! Tell the users what we are doing... though this part should be rather quick.
-  !  CALL WrScr( ' Opening InflowWind input file:  '//InputFileName )
-
+   IF (ErrStat >= AbortErrLev) RETURN
 
-   !-------------------------------------------------------------------------------------------------
-   ! File header
-   !-------------------------------------------------------------------------------------------------
+   !---------------------------------------------------------------------------------------------
+   ! General settings with wind type, direction and output point list (applies to all wind types)
+   !---------------------------------------------------------------------------------------------
 
-   CALL ReadCom( UnitInput, InputFileName, 'InflowWind input file header line 1', TmpErrStat, TmpErrMsg )
+   CurLine = 4
+   CALL ParseVar( InFileInfo, CurLine, "Echo", InputFileData%EchoFlag, TmpErrStat, TmpErrMsg )
    CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
-   IF (ErrStat >= AbortErrLev) THEN
-      CALL Cleanup()
-      RETURN
-   END IF
+   IF (ErrStat >= AbortErrLev) RETURN
 
-   CALL ReadCom( UnitInput, InputFileName, 'InflowWind input file header line 2', TmpErrStat, TmpErrMsg )
+   CALL ParseVar( InFileInfo, CurLine, "WindType", InputFileData%WindType, TmpErrStat, TmpErrMsg )
    CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
-   IF (ErrStat >= AbortErrLev) THEN
-      CALL Cleanup()
-      RETURN
-   END IF
+   IF (ErrStat >= AbortErrLev) RETURN
 
-   CALL ReadCom( UnitInput, InputFileName, 'InflowWind input file separator line', TmpErrStat, TmpErrMsg )
+   CALL ParseVar( InFileInfo, CurLine, "PropagationDir", InputFileData%PropagationDir, TmpErrStat, TmpErrMsg )
    CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
-   IF (ErrStat >= AbortErrLev) THEN
-      CALL Cleanup()
-      RETURN
-   END IF
-
-
-     ! Echo Input Files.
+   IF (ErrStat >= AbortErrLev) RETURN
 
-   CALL ReadVar ( UnitInput, InputFileName, InputFileData%EchoFlag, 'Echo', 'Echo Input', TmpErrStat, TmpErrMsg )
+   CALL ParseVar( InFileInfo, CurLine, "NWindVel", InputFileData%NWindVel, TmpErrStat, TmpErrMsg )
    CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
-   IF (ErrStat >= AbortErrLev) THEN
-      CALL Cleanup()
-      RETURN
-   END IF
-
-      ! If we are Echoing the input then we should re-read the first three lines so that we can echo them
-      ! using the NWTC_Library routines.  The echoing is done inside those routines via a global variable
-      ! which we must store, set, and then replace on error or completion.
-
-   IF ( InputFileData%EchoFlag ) THEN
-
-      CALL OpenEcho ( UnitEcho, TRIM(EchoFileName), TmpErrStat, TmpErrMsg )
-      CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
```

Comment:
The new input file parsing here ignores this `EchoFlag`.

Reply:
Interesting.  I'll take a look at it.  Not sure what I broke there.

Reply:
Don't change anything, yet, though. I'm trying to merge this (from dev) into Jason's FAST.Farm branch and have to fix some merge conflicts. I might change some of this while I am fixing those conflicts.

Reply:
Huh.  Looks like that is completely missing. I'll fix it.

Reply:
Don't give me more merge conflicts!!!! :)

Reply:
In my branch, I am now passing the `EchoFileName` to `InflowWind_ParseInputFileInfo`. I'll let you fix the parts missing the echo file inside the routine.

Reply:
Perfect.  Which branch?  I just setup a branch with a fix in that routine -- can't test it at the moment as I broke my compiler setup... :(
https://github.com/andrew-platt/openfast/commit/578d90c17e848da93bd3a76d7433069c838ee9d4

Reply:
Thanks! I'm working on my f/fast-farm branch, but haven't pushed the changes you'll need, yet, as there are still several broken things with the InflowWind conflicts. I can grab your changes when I'm finished with this.

Reply:
Fixed a few typos in my branch.  Ended up passing the filename also (I find it useful to include that info in the echo file).